### PR TITLE
Add a note about direct test module invocation

### PR DESCRIPTION
**Pull request by sobolevn**
Original BPO: https://bugs.python.org/issue46425

Refs https://github.com/python/cpython/pull/30800 and https://github.com/python/cpython/pull/30799

CC @jaraco 🙂

### CODE REVIEW

**Review by jaraco — APPROVED** [review]
Very nice. Thanks!

File: runtests.rst [hunk 1]
```diff
@@ -50,6 +50,19 @@ path to the test case::
...
+Some test modules also support direct invocation, 
+which might be useful for IDEs and local debugging::
+
+   ./python Lib/test/test_typing.py
+
+But, there are several important notes:
+
+1. This way of running tests exists only 
+   for local developer needs and is discouraged for anything else
+2. Some modules do not support it at all, for example ``test_importlib``,
```
Comment (Mariatta) [inline runtests.rst:62]:
```suggestion
2. Some modules do not support it at all. One example is``test_importlib``.
```

File: runtests.rst [hunk 2]
```diff
@@ -50,6 +50,19 @@ path to the test case::
...
+   in other words: if some module does not have ``unittest.main()``
```
Comment (Mariatta) [inline runtests.rst:63]:
```suggestion
   In other words: if some module does not have ``unittest.main()``, then
```

File: runtests.rst [hunk 3]
```diff
@@ -50,6 +50,19 @@ path to the test case::
...
+   most likely it does not support direct invocation
```
Comment (Mariatta) [inline runtests.rst:64]:
```suggestion
   most likely it does not support direct invocation.
```

**Review by Mariatta — COMMENTED** [review]
Just some grammar improvements.

**Comment by sobolevn** [comment]
Thank you, @Mariatta! 👍

**Review by raghavthind2005 — APPROVED** [review]

**Pull request merged**